src/actions/guides/frontend/internationalization.cr

```diff
@@ -0,0 +1,341 @@
+    end
+    ```
+
+    ```
+    # src/pages/me/show_page.cr
+    class Me::ShowPage < MainLayout
+      def content
+        h1 I18n.t("me.profile", @translator.lang)
+        h3 "\#{I18n.t("me.email", @translator.lang)}:  \#{@current_user.email}"
+        # ...
+      end
+
+      private def helpful_tips
+        h3 "\#{I18n.t("me.next", @translator.lang)}:"
+        ul do
+          # ...
+          li "\#{I18n.t("me.modify_page", @translator.lang)}: src/pages/me/show_page.cr"
+          li "\#{I18n.t("me.after_signin", @translator.lang)}: src/actions/home/index.cr"
+        end
+      end
+
+      private def link_to_authentication_guides
+        link I18n.t("me.auth_guides", @translator.lang), to: "https://luckyframework.org/guides/authentication"
+      end
+    end
+    ```
+
+    ```
+    # src/pages/password_reset_requests/new_page.cr
+    class PasswordResetRequests::NewPage < AuthLayout
+      # ...
+      def content
+        h1 I18n.t("auth.reset_pwd_request", @translator)
+        # ...
+      end
+
+      private def render_form(op)
+        form_for PasswordResetRequests::Create do
+          # ...
+          submit I18n.t("auth.reset_pwd_request", @translator), flow_id: "request-password-reset-button"
+        end
+      end
+    end
+    ```
+
+    ```
+    # src/pages/password_resets/new_page.cr
+    class PasswordResets::NewPage < AuthLayout
+      # ...
+      def content
+        h1 I18n.t("auth.update_pwd", @translator)
+        render_password_reset_form(@operation)
+      end
+
+      private def render_password_reset_form(op)
+        form_for PasswordResets::Create.with(@user_id) do
+          # ...
+          submit I18n.t("auth.update_pwd", @translator), flow_id: "update-password-button"
+        end
+      end
+    end
+    ```
+
+    ```
+    # src/pages/sign_ins/new_page.cr
+    class SignIns::NewPage < AuthLayout
+      # ...
+      def content
+        h1 I18n.t("auth.sign_in", @translator)
+        # ...
+      end
+
+      private def render_sign_in_form(op)
+        form_for SignIns::Create do
+          # ...
+          submit I18n.t("auth.sign_in", @translator), flow_id: "sign-in-button"
+        end
+        link I18n.t("auth.reset_pwd_request", @translator), to: PasswordResetRequests::New
+        text " | "
+        link I18n.t("auth.sign_up", @translator), to: SignUps::New
+      end
+      # ...
+    end
+    ```
+
+    ```
+    # src/pages/sign_ups/new_page.cr
+    class SignUps::NewPage < AuthLayout
+      # ...
+      def content
+        h1 I18n.t("auth.sign_up", @translator)
+        # ...
+      end
+
+      private def render_sign_up_form(op)
+        form_for SignUps::Create do
+          # ...
+          submit I18n.t("auth.sign_up", @translator), flow_id: "sign-up-button"
+        end
+        link I18n.t("auth.sign_in", @translator), to: SignIns::New
+      end
+      # ...
+    end
+    ```
+
+    _Thanks to @paulcsmith & @bdtomlin and others for helping and guiding this document._
```

Comment:
I agree, these are some awesome people; however, we don't really credit any on any of the other guides. My recommendation would be to omit this, but I'll leave that final word to @paulcsmith  😄

Reply:
I removed - but glad too add it back too

Reply:
@jwoertink - can you also look at the modular approach and see what approach you think is better (class or module) at the PR: https://github.com/luckyframework/website/pull/209

Thanks for the review